src/adapters/proposals-query.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2017 The "mysteriumnetwork/mysterium-vpn" Authors.
```

Comment:
No scope of this PR, but from date and name for license `mysteriumnetwork/mysterium-vpn`
- [ ] Create infra ticket "CI script to check_license"